nmmo/systems/ai/move.py

```diff
@@ -1,15 +1,12 @@
 # pylint: disable=R0401
-
-import random
-
 from nmmo.core import action
 from nmmo.systems.ai import utils
 
 
-def random_direction():
-  return random.choice(action.Direction.edges)
+def random_direction(np_random):
```

Comment:
it's pretty annoying to have to pass np_random to all these functions. what do you think about making a MoveHelper() class, and putting all the functions in there? that way it can use self._np_random. 

Not a big deal, but might be nicer.

Reply:
I initially tried it, but I realized it was a whole refactor on both npcs and scripted agents. I will put it on the back burner and tackle it later.